packages/tokens/src/Spacing.tsx

```diff
@@ -0,0 +1,9 @@
+export const spacing = {
+  1: '4px',
+  2: '8px',
+  3: '16px',
+  4: '24px',
+  5: '32px',
+  6: '64px',
+  7: '88px',
```

Comment:
@brookescarlett @hswolff Any thoughts around providing aliases for these? I think having a human-readable name for each of these (`small`, `large`, etc) could help map design specs using spacers to the appropriate value. Providing those both would also allow for rapid iteration through these values.

Reply:
Oh, I suppose the other thing is: We might consider providing these as numbers instead of strings for added flexibility. Not convinced that's right though.

Reply:
Personally, I feel more passionately about only exposing one way to access this rather than what the keys themselves are called.

Reply:
I also think that because these are being used for layout it makes sense to include the pixels but thats just an opinion and open to dropping them